### ISSUE TITLE
Compute global / hemispheric integrated quantities.

### ISSUE THREAD

**Issue by dabail10** [NCAR/CUPiD#45]
This is something the CVDP does. It creates global and hemispheric integrated quantities including:

- globally averaged temperature
- NH sea ice area
- SH sea ice area

others?

For the sea ice we need:

- NH / SH total sea ice extent (area where concentration is >= 15%)
- NH / SH total sea ice area
- NH / SH total sea ice volume
- NH / SH total snow (on sea ice) volume

This uses tarea from the sea ice model (constant gridcell area from sea ice history files or grid files).

Convert aice to fraction if it is not already.

NH extent = where(aice.ge.0.15,tarea,0.) where tlat > 0.
NH area = sum(aice*tarea) for tlat > 0. x 1.0e-12
NH ice volume = sum(hi*tarea) for tlat > 0. x 1.0e-13
NH snow volume = sum(hs*tarea) for tlat > 0. x 1.0e-13

**Comment by dabail10** [comment]
Is this already in python? @nusbaume @justin-richling

Does the ADF call the CVDP?

**Comment by dabail10** [comment]
Here is some python code. Assuming that you have sea ice files uploaded:

ds_area = (ds.TAREA*ds.aice).where(TLAT>0.).sum(dim=['ni','nj'])

**Comment by justin-richling** [comment]
@dabail10 Yeah, we can call the CVDP from the ADF

**Comment by dabail10** [comment]
Aha. I see that the ADF calls the CVDP NCL scripts.

**Comment by justin-richling** [comment]
Yeah, I should've mentions that. It runs in the background of the ADF via NCL scripts. I'm working with Adam Phillips to eventually get the major parts of the CVDP and CVDP-LE into python.

**Comment by phillips-ad** [comment]
@dabail10 the current plan is for a python-only CVDP release to happen in the latter half of 2024. That release will not have all the current CVDP/CVDP-LE capabilities in it. I will speak with you at a later date about what sea ice metrics we should carry over into the new python CVDP package.

**Comment by dabail10** [comment]
Got it. I still think we should work on this one step asap. I will prototype something in python.

**Comment by dabail10** [comment]
@klindsay28 has a number of functions for doing this type of calculation.
